test/ITensorAutoHOOT/runtests.jl

```diff
@@ -1,4 +1,205 @@
 using ITensorNetworkAD
-using Test
+using AutoHOOT
+using ITensors
+using Zygote
 
-@testset "ITensorAutoHOOT.jl" begin end
+const go = AutoHOOT.graphops
+const itensorah = ITensorNetworkAD.ITensorAutoHOOT
+
+@testset "test interface" begin
+  i = Index(2, "i")
+  j = Index(3, "j")
+  k = Index(2, "k")
+  l = Index(4, "l")
+
+  A = randomITensor(i, j)
+  B = randomITensor(j, k)
+  C = randomITensor(k, l)
+
+  out = A * B * C
+  network = [A, C, B]
+
+  nodes, dict = itensorah.generate_einsum_expr([network])
+  network = itensorah.extract_network(nodes[1], dict)
+  out2 = network[1] * network[2] * network[3]
+  @test isapprox(storage(out), storage(out2))
```

Comment:
You should be able to use `isapprox` directly on the ITensors.

Comparing the storage may fail if the tensors are permutations of each other.